src/ci/docker/host-x86_64/x86_64-gnu/Dockerfile

```diff
@@ -17,14 +17,29 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   pkg-config \
   xz-utils \
   mingw-w64 \
+  # libgccjit dependencies
+  flex \
+  libmpfr-dev \
+  libgmp-dev \
+  libmpc3 \
+  libmpc-dev \
   && rm -rf /var/lib/apt/lists/*
 
 COPY scripts/sccache.sh /scripts/
 RUN sh /scripts/sccache.sh
 
+# Fix rustc_codegen_gcc lto issues.
+ENV GCC_EXEC_PREFIX="/usr/lib/gcc/"
```

Comment:
I see that this gets setup in docker/run.sh as well with ENABLE_GCC_CODEGEN. Why is it duplicated here?

Reply:
This was copied from the x86_64-gnu-llvm-17 image but that's no longer necessary. We now use CI rustc by default for all non-dist builders, so there's no need to replace the x86_64-gnu-llvm-17 image with x86_64-gnu image.